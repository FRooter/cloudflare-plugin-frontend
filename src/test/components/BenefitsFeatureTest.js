import React from 'react';
import BenefitsFeature from '../../components/BenefitsFeature/BenefitsFeature';
import renderer from 'react-test-renderer';

it('renders enabled correctly', () => {
  const component = renderer
    .create(
      <BenefitsFeature
        imgSrc="imgSrc"
        title="title"
        description="description"
      />
    )
    .toJSON();
  expect(component).toMatchSnapshot();
});
