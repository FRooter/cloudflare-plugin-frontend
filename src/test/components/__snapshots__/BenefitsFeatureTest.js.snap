// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders enabled correctly 1`] = `
<div
  style={
    Object {
      "padding": "30px 15px 30px 15px",
    }
  }
>
  <img
    src="imgSrc"
    style={
      Object {
        "display": "block",
        "height": "auto",
        "margin": "0 auto",
        "maxWidth": "100px",
        "width": "75px",
      }
    }
  />
  <span
    href="#"
    style={
      Object {
        "color": "#333333",
        "display": "block",
        "fontSize": "16px",
        "padding": "20px 0 0 0",
        "textAlign": "center",
        "width": "100%",
      }
    }
  >
    title
  </span>
  <p
    style={
      Object {
        "color": "#9A9D9E",
        "fontSize": "12px",
        "padding": "10px 0 0 0",
        "textAlign": "center",
      }
    }
  >
    description
  </p>
</div>
`;
